Gruntfile.js

```diff
@@ -60,7 +60,7 @@ module.exports = function (grunt) {
 				requireSpaceBeforeObjectValues: true,
 				validateIndentation: '\t',
 				validateLineBreaks: 'LF',
-				validateQuoteMarks: true,
+				validateQuoteMarks: {mark: '\'', escape: false},
```

Comment:
I don't think `validateQuoteMarks: {mark: '\'', escape: false}` is any different of `validateQuoteMarks: true`. I understand the [Conascence](https://en.wikipedia.org/wiki/Connascence_(computer_programming)) is big for the `validateQuoteMarks: true` and we should probably improve it. However, if such change is not necessary right now, it can be discussed in a separate PR since this one is just about the octal issue.